components/library/glib/Makefile

```diff
@@ -16,34 +16,28 @@
 
 BUILD_BITS = 32_and_64
 BUILD_STYLE = meson
+USE_COMMON_TEST_MASTER= no
```

Comment:
Why do you need this when both `results-32.master` and `results-64.master` are same?